.github/workflows/publish-prerelease.yml

```
@@ -0,0 +1,81 @@
+        run: npm run test:ci
+
+      - name: Configure git
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "GitHub Actions pre-release workflow"
+
+      - name: Set pre-release version
+        run: |
+          # Get current version from package.json
+          CURRENT_VERSION=$(node -p "require('./package.json').version")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)